### ISSUE TITLE
Unable to reproduce the metric

### ISSUE THREAD

**Issue by blcahk** [dengyang17/PPDPP#4]
PPDPP is an extremely constructive piece of work, and we deeply admire it.

We replicated the first stage SFT (10 epochs) on the ESC dataset using a single A6000 GPU and the hyperparameters from the paper, but the evaluation metrics are significantly lower than the results reported in the paper. We are wondering if there might be any overlooked details in the training process, such as data preprocessing, label mapping, or model implementation.

If you could provide a brief guide for replication or key configuration details, it would be extremely helpful to us. Thank you for your time and support!

<img width="785" height="285" alt="Image" src="https://github.com/user-attachments/assets/38d84bed-8843-4a1e-b84f-b0a9b42a942d" />

<img width="934" height="740" alt="Image" src="https://github.com/user-attachments/assets/20ae65ca-be9e-4f08-b0a2-9d423ac974de" />
